specification/cognitiveservices/data-plane/SpeakerRecognition/Verification/stable/2021-09-05/Verification.json

```diff
@@ -0,0 +1,1148 @@
+      "properties": {
+        "locale": {
+          "$ref": "#/definitions/Locale"
+        }
+      }
+    },
+    "TdProfileInfoList": {
+      "description": "Text-Dependent Speaker profile info list",
+      "properties": {
+        "profiles": {
+          "items": {
+            "$ref": "#/definitions/TdProfileInfo"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "example": "{opaqueUrl}"
+        }
+      }
+    },
+    "TdProfileInfo": {
+      "description": "Text-Dependent Speaker profile info",
+      "properties": {
+        "profileId": {
+          "$ref": "#/definitions/ProfileId"
+        },
+        "locale": {
+          "$ref": "#/definitions/Locale"
+        },
+        "enrollmentStatus": {
+          "$ref": "#/definitions/EnrollmentStatus"
+        },
+        "createdDateTime": {
+          "$ref": "#/definitions/CreatedDateTime"
+        },
+        "lastUpdatedDateTime": {
+          "$ref": "#/definitions/LastUpdatedDateTime"
+        },
+        "enrollmentsCount": {
+          "$ref": "#/definitions/EnrollmentsCount"
+        },
+        "enrollmentsLengthInSec": {
+          "$ref": "#/definitions/EnrollmentsLengthInSec"
+        },
+        "enrollmentsSpeechLengthInSec": {
+          "$ref": "#/definitions/EnrollmentsSpeechLengthInSec"
+        },
+        "remainingEnrollmentsCount": {
+          "$ref": "#/definitions/RemainingEnrollmentsCount"
+        },
+        "modelVersion": {
+          "$ref": "#/definitions/ModelVersion"
+        }
+      }
+    },
+    "TdEnrollmentInfo": {
+      "description": "Text-Dependent Speaker profile enrollment info",
+      "properties": {
+        "profileId": {
+          "$ref": "#/definitions/ProfileId"
+        },
+        "enrollmentStatus": {
+          "$ref": "#/definitions/EnrollmentStatus"
+        },
+        "enrollmentsCount": {
+          "$ref": "#/definitions/EnrollmentsCount"
+        },
+        "enrollmentsLengthInSec": {
+          "$ref": "#/definitions/EnrollmentsLengthInSec"
+        },
+        "enrollmentsSpeechLengthInSec": {
+          "$ref": "#/definitions/EnrollmentsSpeechLengthInSec"
+        },
+        "remainingEnrollmentsCount": {
+          "$ref": "#/definitions/RemainingEnrollmentsCount"
+        },
+        "passPhrase": {
+          "$ref": "#/definitions/PassPhrase"
+        },
+        "audioLengthInSec": {
+          "type": "number",
+          "description": "This enrollment audio length in seconds.",
+          "example": 1.83
+        },
+        "audioSpeechLengthInSec": {
+          "type": "number",
+          "description": "This enrollment audio pure speech (which is the amount of audio after removing silence and non-speech segments) length in seconds.",
+          "example": 1.35
+        }
+      }
+    },
+    "TiProfileInfoList": {
+      "description": "Text-Independent Speaker profile info list",
+      "properties": {
+        "profiles": {
```

Comment:
The Azure API Guidelines say that this property should be named `value`. And whether or not you change the name, you should list this property as `required`.

Reply:
done